### DISCUSSION TITLE
Create FlashCards for englisch words

### DISCUSSION THREAD

**Discussion by ThreeCO** [memoetapp/memoet#53] [Show and tell]
Since there is no feature to share Decks yet I tried to copy a nice script to generate english-english flash cards.
You'll need a list of english words. These words will be looked up on https://vocabulary.com.

A flash cards is composed of
* respective word (front)
* part of speech (back)
* definition (back)

The script creates two flash cards: word as question // word as answer

Have fun using it :) 

https://github.com/ThreeCO/wordlist2memoet

**Comment by manhtai** [comment]
What do you mean by "share Decks" feature? Will it be like #38?

**Reply by ThreeCO** [reply]
I assume that we interpret **exporting** and **sharing** as same features. so: yes.
In theory exporting of decks would enable us to back up the decks and/or share em, if they are exported as csv :)

**Reply by manhtai** [reply]
Yeah the issue is for just that purpose. In the mean time you can use the API to get all data you want ;)
